### PR TITLE
Update "Managing Python Dependencies"

### PR DESCRIPTION
**Pull request by TeddyBear06**

### CODE REVIEW

File: transcripts/4.3.2 Selecting Quality Packages Part 2.txt [hunk 1]
```diff
@@ -12,39 +12,39 @@
...
+01:56 about how you can write a great README for a github project, 
 01:59 I am covering a number of things here that in my opinion 
-02:01 should go into a great readme, for example, it should talk about 
+02:01 should go into a great README, for example, it should talk about 
 02:05 what the project actually does, how to install it, 
 02:07 some example usage, how someone could set up a development environment,
 02:11 some link to a change log, and then also license and author info, 
 02:16 you can check out the full article in the link that you see here. 
-02:24 Now let's go back to the requests readme file. I said that I'd like to know 
+02:24 Now let's go back to the Fequests README. I said that I'd like to know
```
Comment (mikeckennedy) [inline transcripts/4.3.2 Selecting Quality Packages Part 2.txt:43]: Hi @TeddyBear06 The changes to this file look great. However, this one line seems to have gotten worse. :) I'm sure you mean Requests README but it's Fequests. Could you update this on your side and push it again to your repo to update the PR?

File: transcripts/3.5 My Virtual Environments Workflow.txt [hunk 2]
```diff
@@ -29,12 +29,12 @@
...
 01:54 and then activate the new one. 
 01:56 So I don't have to type deactivate first, and then of course,
-02:00 I could do m work and just deactivate the virtual environment again. 
+02:00 I could do my work and just deactivate the virtual environment again. 
 02:04 So I am using these ae and de shortcuts or aliases to make it a little bit easier 
 02:09 to work with virtual environments from the command line. 
 02:12 And now I am going to show you how those work behind the scenes. 
 02:15 So what I have done here is I've modified the config file from my shell, 
-02:20 I am using bash so this would be bash profile. 
+02:20 I am using bash so this would be ~/.bash_profile
```
Comment (mikeckennedy) [inline transcripts/3.5 My Virtual Environments Workflow.txt:37]: @TeddyBear06 this was a nice catch. These kinds of changes definitely take things to the next level. ;)

File: transcripts/4.3.2 Selecting Quality Packages Part 2.txt (hunk 1 above)
Comment (mikeckennedy) [reply transcripts/4.3.2 Selecting Quality Packages Part 2.txt:43]: Actually, this is the only thing. I'll just merge and fix it. ;)

Comment (TeddyBear06) [reply transcripts/4.3.2 Selecting Quality Packages Part 2.txt:43]: Sure, my mistake :-)

Comment (TeddyBear06) [reply transcripts/4.3.2 Selecting Quality Packages Part 2.txt:43]: Done ;-)

**Comment by mikeckennedy** [comment]
Just pushed this to the server and it's live. Thanks!

File: transcripts/3.5 My Virtual Environments Workflow.txt (hunk 2 above)
Comment (TeddyBear06) [reply transcripts/3.5 My Virtual Environments Workflow.txt:37]: @mikeckennedy Great :-)

**Pull request merged**